.github/workflows/site.yml

```diff
@@ -86,20 +86,23 @@ jobs:
           path: .ci-temp/checkstyle
           fetch-depth: 0
 
-      - name: Setup local maven cache
-        uses: actions/cache/restore@v4
+      - name: Cache Maven Wrapper
+        uses: actions/cache@v4
```

Comment:
We should not touch maven repository cache, where all jrs are stored 
It is different from maven wrapper cache